src/main/java/org/jabref/JabRefGUI.java

```diff
@@ -104,6 +104,13 @@ private void openWindow(Stage mainStage) {
             }
         });
         Platform.runLater(this::openDatabases);
+
+        if (!(Globals.getFileUpdateMonitor().isActive())) {
+            this.getMainFrame().getDialogService()
+                    .showErrorDialogAndWait(Localization.lang("Unable to monitor file changes. Please close files " +
```

Comment:
I think, this has to go in one line so that the localization consistency check works. --> We have all keys in `Localiazation_en.properties` and a check checking all `.java` files whether the key set in the property file matches the language keys used in the code.

Reply:
Just run LocalizationConsistencyTest under l10n

Reply:
I added this to Localization_en.properties and LocalizationConsistencyTest passes all 49 checks.